### PR TITLE
Fixes exclude options

### PR DESCRIPTION
**Pull request by tinganho**
**Fixes issue:** #7126

### CODE REVIEW

File: src/compiler/tsc.ts [hunk 1]
```diff
@@ -710,14 +710,16 @@ namespace ts {
...
-                exclude: ["node_modules"]
+                compilerOptions: serializeCompilerOptions(compilerOptions)
             };
 
             if (fileNames && fileNames.length) {
                 // only set the files property if we have at least one file
                 configurations.files = fileNames;
             }
+            else {
+                configurations.exclude = ["node_modules"];
```
Comment (mhegazy) [inline src/compiler/tsc.ts]: can you add outDir as well?

Comment (tinganho) [inline src/compiler/tsc.ts]: Sure, what should be the value?

**Comment by mhegazy** [comment]
if outDir is specified, we should add it to the excludes. if not, nothing should be added.

**Comment by tinganho** [comment]
Ok I submit a new PR tomorrow. Don't have time today.

**Pull request merged**